pkg/config/config_template.yaml

```diff
@@ -217,7 +218,40 @@ api_key:
 #
 # The timeout to execute the command in second
 # secret_backend_timeout: 5
-
+#
+{{ end -}}
+{{- if .LogsAgent }}
+# Logs agent
+#
+# Enable logs collection for all containers, disabled by default
+# logs_enabled: false
+#
+# logs_config:
+#
+# Enable container log collection for all the containers (see ac_exclude to filter out containers)
+#   container_collect_all: false
+#
+# Define the endpoint and port to hit when using a proxy for logs. The logs are forwarded in TCP
+# therefore the proxy must be able to handle TCP connections.
+#   logs_dd_url: <endpoint>:<port>
+#
+# Disable the SSL encryption (default to false). This parameter should only be used when logs are
+# forwarded locally to a proxy. It is highly recommended to then handle the SSL encryption 
+# on the proxy side. 
+#   logs_no_ssl: false
+#
+# Global processing rules that are applied to all the logs. The available rules are
+# "exclude_at_match", "include_at_match" and "mask_sequences"
```

Comment:
are these rules self-explanatory, or should we refer users to additional docs?

Reply:
updated